packages/components/src/VirtualizedList/NoRows/NoRows.scss

```diff
@@ -0,0 +1,8 @@
+.no-result {
+	flex-grow: 1;
+	flex-shrink: 1;
```

Comment:
Those 2 properties are not used. Its container is not a display: flex.
Remove them please